script/ci/configure.sh

```
@@ -66,6 +67,37 @@ if [[ "$compiler_name" == "gcc" || "$compiler_name" == "clang" ]]; then
         fi
     fi
 
+    if [[ "$APCI_ONEAPI" != 0 ]]; then
+        load_variable_if_not_exist ONEAPI_PATH
+
+        # shellcheck source=script/ci/install/oneapi/setvars.sh
+        source "${APCI_ALPAKA_ROOT}/script/ci/install/oneapi/setvars.sh"
```

Shellcheck (0.11.0)

[info] Not following: script/ci/install/oneapi/setvars.sh was not specified as input (see shellcheck -x).

(SC1091)